web/public/css/components/cd-button.less

```diff
@@ -0,0 +1,22 @@
+.btn {
+	-webkit-border-radius: 4px !important;
+	-moz-border-radius: 4px !important;
+	border-radius: 4px !important;
```

Comment:
You can get rid of these now, I got rid of the * { border-radius: 0 !important; } rule. Round corners on ALL THE THINGS